app/lib/common/widgets/redact_content.dart

```diff
@@ -113,30 +107,19 @@ class RedactContentWidget extends ConsumerWidget {
         );
       }
 
-      if (ctx.mounted) {
-        Navigator.of(ctx, rootNavigator: true).pop();
-        Navigator.of(ctx, rootNavigator: true).pop(true);
-        customMsgSnackbar(ctx, L10n.of(ctx).contentSuccessfullyRemoved);
-        if (onSuccess != null) {
-          onSuccess!();
-        }
+      EasyLoading.dismiss();
+      if (!ctx.mounted) return;
+      EasyLoading.showToast(L10n.of(ctx).contentSuccessfullyRemoved);
+      if (onSuccess != null) {
+        onSuccess!();
       }
     } catch (e) {
-      if (ctx.mounted) {
-        Navigator.of(ctx, rootNavigator: true).pop();
-        showAdaptiveDialog(
-          context: ctx,
-          builder: (ctx) => DefaultDialog(
-            title: Text('${L10n.of(ctx).redactionFailed} $e'),
-            actions: <Widget>[
-              OutlinedButton(
-                onPressed: () => Navigator.of(ctx, rootNavigator: true).pop(),
-                child: Text(L10n.of(ctx).close),
-              ),
-            ],
-          ),
-        );
-      }
+      EasyLoading.dismiss();
+      if (!ctx.mounted) return;
+      EasyLoading.showError(
```

Comment:
I've seen this pattern a few times now, but effectively you don't need to `dismiss` if you are showing another message right after. that was necessary with the navigators but is not necessary with EasyLoading and make the code unnecessarily verbose. How about instead:

```suggestion
      if (!ctx.mounted) {
        EasyLoading.dismiss();
        return;
      }
      EasyLoading.showError(
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/e46df3fe7c86d70f46d4f2893e30dba77db5953d